main.cpp

```diff
@@ -411,32 +356,111 @@ string deleteSpaceInString (string name){
     return output;
 }
 
-static int login_callback(void *unused, int count, char **data, char **columns){                   
 
-    cout << "name: " << GlobalName << endl;
-    cout << "password " << GlobalPassword << endl;
-    if (GlobalName == data[0]){
-        printf("%s is in table \n", data[0]);
-        printf("%s is password \n", data[1]);
-        if (GlobalPassword == data[1]){
-            cout << "login succeffuly" << endl;
-            loginSucceffuly = true;
-        }
-        else{
-            cout << "Inccorect password" << endl;
-        }
+bool loginUser(string inputName, string inputPassword) {
+    string name = deleteSpaceInString(inputName);
+    string password = deleteSpaceInString(inputPassword);
+
+    sqlite3 *db;
+    char *errMsg = nullptr;
+    string sql = "INSERT INTO USERS(username, password) VALUES('"+ name +"', '"+ password +"');";
+
+    int rc = sqlite3_open(WAYTODB, &db);
+
+    USER user;
+    user.name = name;
+    user.password = password;
+
+    rc = sqlite3_exec(db, sql.c_str(), NULL, NULL, NULL);
+    
+    if (rc != SQLITE_OK ){
+        sqlite3_exec(db, "SELECT * FROM USERS", login_callback, &user, NULL);
+        fprintf(stderr, "SQL error: %s\n", errMsg);
+        sqlite3_free(errMsg);
+    } 
+    else{
```

Comment:
Use spaces between operators
```suggestion
    else {
```